clients/ui/frontend/src/app/pages/modelCatalog/components/ValidatedModelCard.tsx

```diff
@@ -0,0 +1,247 @@
+import React from 'react';
+import {
+  Badge,
+  Button,
+  Card,
+  CardBody,
+  CardFooter,
+  CardHeader,
+  CardTitle,
+  Flex,
+  FlexItem,
+  Tooltip,
+  Content,
+  ContentVariants,
+  Skeleton,
+} from '@patternfly/react-core';
+import text from '@patternfly/react-styles/css/utilities/Text/text';
+import { ChartBarIcon, HelpIcon, ChevronLeftIcon, ChevronRightIcon } from '@patternfly/react-icons';
+import { Link } from 'react-router-dom';
+import {
+  CatalogModel,
+  CatalogSource,
+  CatalogPerformanceMetricsArtifact,
+  CatalogAccuracyMetricsArtifact,
+} from '~/app/modelCatalogTypes';
+import { getModelName } from '~/app/pages/modelCatalog/utils/modelCatalogUtils';
+import { catalogModelDetailsFromModel } from '~/app/routes/modelCatalog/catalogModel';
+import { getLabels } from '~/app/pages/modelRegistry/screens/utils';
+import { extractValidatedModelMetrics } from '~/app/pages/modelCatalog/utils/validatedModelUtils';
+import ModelCatalogLabels from '~/app/pages/modelCatalog/components/ModelCatalogLabels';
+
+type ValidatedModelCardProps = {
+  model: CatalogModel;
+  source: CatalogSource | undefined;
+  performanceMetrics?: CatalogPerformanceMetricsArtifact;
+  accuracyMetrics?: CatalogAccuracyMetricsArtifact;
+  truncate?: boolean;
+};
+
+const ValidatedModelCard: React.FC<ValidatedModelCardProps> = ({
+  model,
+  source,
+  performanceMetrics,
+  accuracyMetrics,
+  truncate = false,
+}) => {
+  const allLabels = model.customProperties ? getLabels(model.customProperties) : [];
+  const validatedLabels = allLabels.includes('validated') ? ['validated'] : [];
+
+  const metrics = extractValidatedModelMetrics(performanceMetrics, accuracyMetrics);
+
+  return (
+    <Card
+      isFullHeight
+      data-testid="validated-model-catalog-card"
+      key={`${model.name}/${model.source_id}`}
+    >
+      <CardHeader>
+        <CardTitle>
+          <Flex alignItems={{ default: 'alignItemsCenter' }} style={{ paddingLeft: '8px' }}>
+            {model.logo ? (
+              <img
+                src={model.logo}
+                alt="model logo"
+                style={{ height: '56px', width: '56px', marginRight: '8px' }}
+              />
+            ) : (
+              <Skeleton
+                shape="square"
+                width="56px"
+                height="56px"
+                screenreaderText="Brand image loading"
+                style={{ marginRight: '8px' }}
+              />
+            )}
+            <FlexItem align={{ default: 'alignRight' }}>
+              <Badge className="pf-m-purple">Validated</Badge>
+            </FlexItem>
```

Comment:
Let's still use the Label component as the existing card does so they have the same shape/size, the only difference for validated models is we'll have it say Validated instead of the source name and be purple (Label has a `color="purple"` prop as well).